frontend/src/components/pages/RegistrantExperience/Waiver/index.tsx

```diff
@@ -1,13 +1,136 @@
 import React from "react";
-import { Box, Text } from "@chakra-ui/react";
+import {
+  Box,
+  Text,
+  VStack,
+  Checkbox,
+  ListItem,
+  OrderedList,
+  Stack,
+  Radio,
+  RadioGroup,
+  FormControl,
+  FormLabel,
+  Input,
+} from "@chakra-ui/react";
+import { ReactJSXElement } from "@emotion/react/types/jsx-namespace";
+import {
+  OptionalClauseResponse,
+  RequiredClauseResponse,
+  WaiverActions,
+  WaiverInterface,
+} from "./waiverTypes";
+
+interface WaiverPageProps {
+  waiverInterface: WaiverInterface;
+  waiverDispatch: any;
+}
+
+const WaiverPage = ({
+  waiverInterface,
+  waiverDispatch,
+}: WaiverPageProps): React.ReactElement => {
+  const optionalClausesJSX: ReactJSXElement[] = Array.from(
+    waiverInterface.optionalClauses,
+  ).map((clause: OptionalClauseResponse, index: number) => (
+    <VStack spacing={6} pt="2" align="stretch" key={index}>
+      <Text>{clause.text}</Text>
+      <RadioGroup
+        onChange={() =>
+          waiverDispatch({
+            type: WaiverActions.CLICK_OPTIONAL_CLAUSE,
+            payload: clause,
+          })
+        }
+        key={index}
+        value={String(clause.agreed)}
+      >
+        <Stack spacing={5} direction="row">
+          <Radio value="true" mb="0">
+            {/* NOTE!?: I use mb='0' because there's a margin misalignment issue with Radio component */}
+            I agree
+          </Radio>
+          <Radio value="false">
+            I{" "}
+            <Text as="span" fontWeight="bold">
+              do not
+            </Text>{" "}
+            agree
+          </Radio>
+        </Stack>
+      </RadioGroup>
+    </VStack>
+  ));
+
+  const requiredClausesJSX: ReactJSXElement[] = Array.from(
+    waiverInterface.requiredClauses,
+  ).map((clause: RequiredClauseResponse, index: number) => (
+    <ListItem key={index}>{clause.text}</ListItem>
+  ));
 
-const Waiver = (): React.ReactElement => {
   return (
     <Box>
-      <Text textStyle="displayXLarge">Camp Registration</Text>
-      <Text>Waiver</Text>
+      <VStack spacing={8} pt="1" align="stretch">
+        <Text textStyle="displayXLarge">Camp Registration</Text>
+        <Text textStyle="displayLarge">{waiverInterface.campName}</Text>
+        <Text textStyle="displayLarge" color="primary.green.100">
+          Waiver
+        </Text>
+        <Text textStyle="displayMedium" fontWeight="bold">
+          In consideration of the participation of my child/children, (the
+          “child or children“), in the Focus on Nature photography camp/workshop
+          and all activities associated therewith,
+        </Text>
+        <Text fontWeight="bold">
+          I, the undersigned parent/guardian of the child/children, agree to the
+          following terms and conditions:
```

Comment:
Think these should be the same paragraph + text styling?

<img width="1423" alt="Screen Shot 2022-11-26 at 3 48 23 AM" src="https://user-images.githubusercontent.com/40974372/204080517-37b3b36e-3c15-4479-b030-83bc2a4a10f1.png">

Reply:
Yep, thanks for the catch